facade/src/main/java/org/jboss/pnc/facade/rsql/RSQLProducer.java

```diff
@@ -40,6 +41,16 @@ public interface RSQLProducer {
      */
     <DB extends GenericEntity<?>> Predicate<DB> getCriteriaPredicate(Class<DB> type, String rsql);
 
+    /**
+     * Analogical to {@link this#getCriteriaPredicate(Class, String)}. Unlike the previous method, it uses only this
+     * single mapper during the whole computation.
```

Comment:
what if the mapper provided would reference other mappers? How does it guarantee that no other mappers are used?
fix the javadoc.